scalyr_agent/configuration.py

```diff
@@ -223,6 +223,53 @@ def parse(self):
                 if https_server != server:
                     self.__config["scalyr_server"] = https_server
 
+            # Set defaults based on `max_send_rate_enforcement` value
+            if (
+                self.__config["max_send_rate_enforcement"] == "legacy"
+                or self.__config["disable_max_send_rate_enforcement_overrides"]
+            ):
+                if "max_allowed_request_size" not in self.__config:
+                    self.__config["max_allowed_request_size"] = 1048576
```

Comment:
To make it easier to read it would be nice to use the NB to bytes notation we used before - ``1 * 1024 * 1024``

Reply:
Same goes for other values.